.github/workflows/deploy-docs.yaml

```
@@ -4,12 +4,16 @@ on:
   push:
     branches:
       - main
+      - beta/*
     paths:
       - mkdocs.yaml
       - "**/*.md"
       - "**/*.svg"
       - "**/*.png"
       - "**/*.jpg"
+    tags:
+      - v*
+      - [0-9]+.[0-9]+*
```

https://github.com/autowarefoundation/autoware/pull/3026#discussion_r1022903658

```suggestion
      - "[0-9]+.[0-9]+*"
```